### PR TITLE
Added ameliorated and exacerbated by conditions

### PR DESCRIPTION
**Pull request by VarunReddy1111**

### CODE REVIEW

File: agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java [hunk 1]
```diff
@@ -355,7 +355,7 @@ private DiseaseDownloadRow getBaseDiseaseDownloadRow(DiseaseAnnotationDocument a
 			for (org.alliancegenome.curation_api.model.entities.DiseaseAnnotation da : annotation.getPrimaryAnnotations()) {
 				if (CollectionUtils.isNotEmpty(da.getConditionRelations())) {
 					List<ConditionRelation> conditionModifiers = da.getConditionRelations().stream().filter(conditionRelation -> conditionRelation.getConditionRelationType() != null)
-						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced")).toList();
+						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced") || conditionRelation.getConditionRelationType().getName().contains("ameliorated") || conditionRelation.getConditionRelationType().getName().contains("exacerbated")).toList();
```
Comment (devopsagr) [inline agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java:358]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 400 characters (found 410).

File: agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java [hunk 2]
```diff
@@ -355,7 +355,10 @@ private DiseaseDownloadRow getBaseDiseaseDownloadRow(DiseaseAnnotationDocument a
 			for (org.alliancegenome.curation_api.model.entities.DiseaseAnnotation da : annotation.getPrimaryAnnotations()) {
 				if (CollectionUtils.isNotEmpty(da.getConditionRelations())) {
 					List<ConditionRelation> conditionModifiers = da.getConditionRelations().stream().filter(conditionRelation -> conditionRelation.getConditionRelationType() != null)
-						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced")).toList();
+						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition")
```
Comment (devopsagr) [inline agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java:358]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace

File: agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java [hunk 3]
```diff
@@ -355,7 +355,10 @@ private DiseaseDownloadRow getBaseDiseaseDownloadRow(DiseaseAnnotationDocument a
 			for (org.alliancegenome.curation_api.model.entities.DiseaseAnnotation da : annotation.getPrimaryAnnotations()) {
 				if (CollectionUtils.isNotEmpty(da.getConditionRelations())) {
 					List<ConditionRelation> conditionModifiers = da.getConditionRelations().stream().filter(conditionRelation -> conditionRelation.getConditionRelationType() != null)
-						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced")).toList();
+						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") 
+						|| conditionRelation.getConditionRelationType().getName().contains("induced")
```
Comment (devopsagr) [inline agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java:359]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace

File: agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java [hunk 4]
```diff
@@ -355,7 +355,10 @@ private DiseaseDownloadRow getBaseDiseaseDownloadRow(DiseaseAnnotationDocument a
 			for (org.alliancegenome.curation_api.model.entities.DiseaseAnnotation da : annotation.getPrimaryAnnotations()) {
 				if (CollectionUtils.isNotEmpty(da.getConditionRelations())) {
 					List<ConditionRelation> conditionModifiers = da.getConditionRelations().stream().filter(conditionRelation -> conditionRelation.getConditionRelationType() != null)
-						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") || conditionRelation.getConditionRelationType().getName().contains("induced")).toList();
+						.filter(conditionRelation -> conditionRelation.getConditionRelationType().getName().contains("has_condition") 
+						|| conditionRelation.getConditionRelationType().getName().contains("induced") 
+						|| conditionRelation.getConditionRelationType().getName().contains("ameliorated")
```
Comment (devopsagr) [inline agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java:360]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace

**Review by markquintontulloch — APPROVED** [review]

**Pull request merged**